src/helpers/ErrorBoundary.tsx

```diff
@@ -0,0 +1,66 @@
+import { ErrorBoundaryFallbackPage } from '@openshift-console/dynamic-plugin-sdk';
+import * as React from 'react';
+export type ErrorBoundaryFallbackProps = {
+  errorMessage: string;
+  componentStack: string;
+  stack: string;
+  title: string;
+};
+
+type ErrorBoundaryProps = React.PropsWithChildren<{
+  FallbackComponent?: React.ComponentType<ErrorBoundaryFallbackProps>;
+}>;
+
+/** Needed for tests -- should not be imported by application logic */
+export type ErrorBoundaryState = {
+  hasError: boolean;
+  error: { message: string; stack: string; name: string };
+  errorInfo: { componentStack: string };
+};
+class ErrorBoundary extends React.Component<ErrorBoundaryProps, ErrorBoundaryState> {
```

Comment:
this is just a copy-paste from `ocp console` right ? Seems like a simple component, can you create a PR to expose it via `plugins-shared` package ?